### PR TITLE
add max claimable tokens query

### PR DESCRIPTION
**Pull request by asalzmann**
## Context and purpose of the change
- Add a query to get the total number of tokens a user can claim in an airdrop period
- Add a setup script to run the airdrop locally (reproducibly) 

## Changelog
- Add `includeClaimed` arg to `TotalClaimable` query (and dependent functions)

## Testing
- Add unittest

I also tested this manually, by running
```
make start-docker
bash scripts/airdrop.sh
$STRIDE_MAIN_CMD query claim total-claimable stride stride1nf6v2paty9m22l3ecm7dpakq2c92ueyununayr "true"
```
and verifying the output
```
coins:
- amount: "600000"
  denom: ustrd
```
using `"false"` returns
```
coins: []
```

### CODE REVIEW

File: scripts/airdrop.sh [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+
+## AIRDROP SETUP
+# Transfer uatom from gaia to stride, so that we can liquid stake later
+$GAIA_MAIN_CMD tx ibc-transfer transfer transfer channel-0 stride1nf6v2paty9m22l3ecm7dpakq2c92ueyununayr 1000000uatom --from ${GAIA_VAL_PREFIX}1 -y 
+sleep 5
+
+
+# Fund the distributor account
+$STRIDE_MAIN_CMD tx bank send val1 stride1z835j3j65nqr6ng257q0xkkc9gta72gf48txwl 600000ustrd --from val1
```
Comment (sampocs) [inline scripts/airdrop.sh:25]:
```suggestion
$STRIDE_MAIN_CMD tx bank send val1 stride1z835j3j65nqr6ng257q0xkkc9gta72gf48txwl 600000ustrd --from val1 -y
```

File: scripts/airdrop.sh [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+sleep 5
+# Fund the airdrop account
+$STRIDE_MAIN_CMD tx bank send val1 stride1nf6v2paty9m22l3ecm7dpakq2c92ueyununayr 1000000000ustrd --from val1
```
Comment (sampocs) [inline scripts/airdrop.sh:28]:
```suggestion
$STRIDE_MAIN_CMD tx bank send val1 stride1nf6v2paty9m22l3ecm7dpakq2c92ueyununayr 1000000000ustrd --from val1 -y
```

File: scripts/airdrop.sh [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+sleep 5
+
+# Create the airdrop, so that the airdrop account can claim tokens
+$STRIDE_MAIN_CMD tx claim create-airdrop stride 1666792900 40000000 ustrd --from distributor-test
```
Comment (sampocs) [inline scripts/airdrop.sh:32]:
```suggestion
$STRIDE_MAIN_CMD tx claim create-airdrop stride 1666792900 40000000 ustrd --from distributor-test -y
```

File: scripts/airdrop.sh [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+sleep 5
+# Set airdrop allocations
+$STRIDE_MAIN_CMD tx claim set-airdrop-allocations stride stride1nf6v2paty9m22l3ecm7dpakq2c92ueyununayr 1 --from distributor-test
```
Comment (sampocs) [inline scripts/airdrop.sh:35]:
```suggestion
$STRIDE_MAIN_CMD tx claim set-airdrop-allocations stride stride1nf6v2paty9m22l3ecm7dpakq2c92ueyununayr 1 --from distributor-test -y
```

File: scripts/airdrop.sh [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+# Check balances before claims
+$STRIDE_MAIN_CMD query bank balances stride1nf6v2paty9m22l3ecm7dpakq2c92ueyununayr
+# NOTE: You can claim here using the CLI, or from the frontend!
+# Claim 20% of the free tokens
+$STRIDE_MAIN_CMD tx claim claim-free-amount --from airdrop-test --gas 400000
+sleep 5
+# Stake, to claim another 20%
+$STRIDE_MAIN_CMD tx staking delegate stridevaloper1nnurja9zt97huqvsfuartetyjx63tc5zrj5x9f 100ustrd --from airdrop-test --gas 400000
+sleep 5
+# Liquid stake, to claim the final 20% of tokens
```
Comment (sampocs) [inline scripts/airdrop.sh:49]: 60%?

**Review by sampocs — COMMENTED** [review]
Lgtm! Airdrop script worked as expected. You mentioned adding a new unit test but I'm not seeing it - was that meant as a TODO?

**Comment by asalzmann** [comment]
@sampocs had forgotten to hit push on the unittest, just pushed it

File: scripts/airdrop.sh (hunk 5 above)
Comment (asalzmann) [reply scripts/airdrop.sh:49]: done

**Pull request merged**